pkg/app/api/api/web_api.go

```diff
@@ -318,6 +319,12 @@ func (a *WebAPI) AddApplication(ctx context.Context, req *webservice.AddApplicat
 		a.logger.Error("failed to extract claims", zap.Error(err))
 		return nil, status.Error(codes.Internal, "internal error")
 	}
+
+	// The path to the application directory must be relative.
+	if strings.HasPrefix(req.GitPath.Path, "/") {
+		return nil, status.Error(codes.InvalidArgument, "the path must be a relative path")
+	}
```

Comment:
Some entities in our database are prefixed by this `/`.

Reply:
That is why some applications were not triggered even if their files were touched.